src/Compilers/VisualBasic/Portable/Symbols/Tuples/TuplePropertySymbol.vb

```diff
@@ -132,5 +132,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Public Overrides Function GetAttributes() As ImmutableArray(Of VisualBasicAttributeData)
             Return Me._underlyingProperty.GetAttributes()
         End Function
+
+        Public Overrides ReadOnly Property IsRequired As Boolean
+            Get
+                Return False
```

Comment:
>Return False [](http://example.com/codeflow?start=16&length=12)

This doesn't look like a justified assumption. The ```_underlyingProperty``` is almost never a source property